app/views/application_records/show.haml

```diff
@@ -1,15 +1,16 @@
-%h1 Application Record
+%h1.h2="#{@record.position.name} Application Record"
```

Comment:
If it's everywhere, then a generic CSS rule on `h1` is in order.

If it's just here, then there two alternatives leap to mind which would communicate our intent a little better:

1. Wrap the page in some kind of div (`.smaller-headers`) and then write CSS specific to that:
  ```css
  .smaller-headers h1{ font-size: ??px; }
  ```
2. Writing a more descriptive CSS class to apply directly to the headings:
  ```css
  h1.we-want-it-to-be-a-little-smaller{ font-size: ??px; }
  ```